src/payment/data/sagas.js

```diff
@@ -19,8 +22,10 @@ import {
   fetchCaptureKey,
 } from './actions';
 
+import { DEFAULT_STATUS } from '../checkout/payment-form/flex-microform/constants';
```

Comment:
nit: I think it would be more clear to explicitly use `STATUS_LOADING` here, it's not abundantly clear what the default is.

Reply:
Good call, changed.